From 006117c1424615b403b32126e05521fd654a5a93 Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Fri, 13 May 2022 17:08:47 +0200
Subject: [PATCH] cli_TEST: Fix compatibility with CLI11 2.0

Signed-off-by: Silvio <silvio@traversaro.it>
---
 cli/src/cli_TEST.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cli/src/cli_TEST.cc b/cli/src/cli_TEST.cc
index 85c8d2b..1e819c9 100644
--- a/cli/src/cli_TEST.cc
+++ b/cli/src/cli_TEST.cc
@@ -86,7 +86,7 @@ std::shared_ptr<TestOptions> addFlags(CLI::App &_app)
   excludesB->excludes(excludesA);
 
   _app.add_option("--default-value", opt->defaultValueOption,
-      "Option with default value", true);
+      "Option with default value")->capture_default_str();
 
   return opt;
 }
